ffmpeg/ffmpeg.go

```diff
@@ -41,6 +41,11 @@ type TranscodeOptions struct {
 	Device  string
 }
 
+type TranscodeResults struct {
```

Comment:
This name is what we agreed on in https://github.com/livepeer/lpms/issues/124 , but given https://github.com/livepeer/lpms/issues/125 I'm wondering if a more generic name like "MediaInfo" would be appropriate

Reply:
I'm in favor of renaming to `MediaInfo`.